### PR TITLE
adds a default clearml-core service account for the core components

### LINKED ISSUE #247: [allegroai/clearml] clearml apiserver needs to run as root but deployment doesn't specify
### Describe the bug a clear and concise description of what the bug is.

When using this helm chart with OpenShift, the clearml apiserver fails with the following log message:

```
mkdir: cannot create directory ‘/var/log/clearml’: Permission denied
```

OpenShift, by default, randomizes the UID for containers that do not specify a user to use in the pod spec. 

Looking inside this container:

```
$ ls -l /var/log/
total 244
-rw-r--r--. 1 root root  12641 Jul 28 07:21 alternatives.log
drwxr-xr-x. 1 root root     60 Jul 28 07:20 apt
-rw-rw----. 1 root utmp      0 Apr 11 00:00 btmp
-rw-r--r--. 1 root root 196255 Jul 28 07:21 dpkg.log
-rw-r--r--. 1 root root   3232 Apr 11 00:00 faillog
-rw-rw-r--. 1 root utmp  29492 Apr 11 00:00 lastlog
drwxr-xr-x. 2 root adm      41 Jul 28 07:22 nginx
-rw-rw-r--. 1 root utmp      0 Apr 11 00:00 wtmp
$ ls -l /var/ 
total 0
drwxr-xr-x. 2 root root    6 Dec  9  2022 backups
drwxr-xr-x. 1 root root   48 Apr 11 00:00 cache
drwxr-xr-x. 1 root root  104 Jul 28 07:21 lib
drwxrwsr-x. 2 root staff   6 Dec  9  2022 local
lrwxrwxrwx. 1 root root    9 Apr 11 00:00 lock -> /run/lock
drwxr-xr-x. 1 root root   70 Jul 28 07:21 log
drwxrwsr-x. 2 root mail    6 Apr 11 00:00 mail
drwxr-xr-x. 2 root root    6 Apr 11 00:00 opt
lrwxrwxrwx. 1 root root    4 Apr 11 00:00 run -> /run
drwxr-xr-x. 2 root root   18 Apr 11 00:00 spool
drwxrwxrwt. 2 root root    6 Dec  9  2022 tmp
drwxr-xr-x. 3 root root   18 Jul 28 07:21 www
$ 
```

given that `/var/log` in this container is not writeable by non-root, it appears that this container _must_ run as root and, therefore should directly specify the UID in the StatefulSet

There are alternatives:
* fix `/var/log` to be writeable by the root _group_
* use a different log location
* many others
Additionally, it would be useful for the apiserver to use a service account. That way the service account can be granted specific security context privileges (anyuid, etc.) more easily. Without a defined service account, the apiserver will run with the namespace default, which likely does not have the privileges to run as a specific UID.

### What's your helm version?

version.BuildInfo{Version:"v3.12.3", GitCommit:"3a31588ad33fe3b89af5a2a54ee1d25bfe6eaa5e", GitTreeState:"clean", GoVersion:"go1.20.7"}

### What's your kubectl version?

Client Version: version.Info{Major:"1", Minor:"23", GitVersion:"v0.23.0", GitCommit:"7c299f197c583559613ae9cdeb01f58496437a29", GitTreeState:"clean", BuildDate:"2022-02-08T22:39:24Z", GoVersion:"go1.17.2", Compiler:"gc", Platform:"linux/amd64"} Server Version: version.Info{Major:"1", Minor:"26", GitVersion:"v1.26.3+b404935", GitCommit:"9ded806a7d5deed41a20f680cf89dae58bbb5697", GitTreeState:"clean", BuildDate:"2023-04-19T02:20:48Z", GoVersion:"go1.19.6", Compiler:"gc", Platform:"linux/amd64"}

### What's the chart version?

7.3.0

### Enter the changed values of values.yaml?

```
elasticsearch:
  enabled: true
  rbac:
    create: true
    serviceAccountName: "clearml-elastic"
  httpPort: 9200
  roles:
    master: "true"
    ingest: "true"
    data: "true"
    remote_cluster_client: "true"
  replicas: 1
  # Readiness probe hack for a single-node cluster (where status will never be green). Should be removed if using replicas > 1
  clusterHealthCheckParams: "wait_for_status=yellow&timeout=1s"
```

### PR DESCRIPTION
**Pull request by thoraxe**
**What this PR does / why we need it**:
This PR adds a default service account declaration to the clearml components (webserver, fileserver, apiserver) and a YAML to the chart to create that SA. 

**Checklist**
- [X] Reviewed the [`CONTRIBUTING.md`](https://github.com/allegroai/clearml-helm-charts/blob/main/CONTRIBUTING.md#pull-requests) guide (**required**)
- [X] Verify the work you plan to merge addresses an existing [issue](https://github.com/allegroai/clearml-helm-charts/issues) (If not, open a new one) (**required**)
- [X] Check your branch with `helm lint` (**required**)
- [X] Update `version` in `Chart.yaml` according [semver](https://semver.org/) rules (**required**)
- [X] Substitute `annotations` section in `Chart.yaml` annotating implementations (useful for Artifecthub changelog) (**required**)
- [x] Update chart README using [helm-docs](https://github.com/norwoodj/helm-docs) (**required**)

**Which issue(s) this PR fixes**:
Fixes #247 

**Special notes for your reviewer**:
It's not entirely a complete PR because it should either do one of two things, which I'm not entirely sure how to do:

1) enable or disable using the ServiceAccount at all
2) create individual SAs for each component based on the name specified in `values.yaml`

### CODE REVIEW

File: charts/clearml/templates/serviceAccount.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: clearml-core
```
Comment (valeriano-manassero) [inline charts/clearml/templates/serviceAccount.yaml:4]: EOF blank line missing

**Review by valeriano-manassero — CHANGES_REQUESTED** [review]
README needs to be synced

**Review by valeriano-manassero — APPROVED** [review]

**Comment by valeriano-manassero** [comment]
@thoraxe I did some changes to reach CI ok status.
Would you mind to check just to be sure this is what you needed in this PR?

**Comment by thoraxe** [comment]
It looks like it does the job to me -- it adds a default service account for the core components with a value. Thanks for cleaning up my mess!

**Pull request merged**